<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inference</class>
 <widget class="QDialog" name="inference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>inference from prototype base:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="prototypeBaseName_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="editPrototypeBase_btn">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Covariate Files:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Inference Property:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="InferedProperty_comboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="CovariateFiles_tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Output Soil File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="OutputSoilFile_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="SoilFileCreate_btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Output Uncertaintiy File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="OutputUncerFile_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Similarity cutoff (threshold):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Threshold_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>RAM Requirement:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RAMEfficient_low_rbtn">
         <property name="text">
          <string>Low</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RAMEfficient_medium_rbtn">
         <property name="text">
          <string>Medium</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RAMEfficient_high_rbtn">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Inference_OK_btn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
